static/docs/get-started/add-files.md

```diff
@@ -1,8 +1,8 @@
-# Add Files
+# Add Data Files
```

Comment:
it's not only data - some people add models, may be should even change the name to include "or Directories"?

Reply:
You're right. Updated.